doc/corewar-glossary.html

```
@@ -0,0 +1,675 @@
+<hr>
+
+<dt><a name="SelfSplitting">Self-Splitting</a></dt>
+
+<dd>Strategy of amplifying the number of processes executing a
+piece of code. 
+
+<pre class="code94">
+    <span class="label">example</span>   ...
+
+              spl   0                  <span class="comment">; self-splitting</span>
+    <span class="label">clear</span>     mov   bomb,  >example
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)